packages/extractor-vue/src/__test__/extractor.test.ts

```diff
@@ -0,0 +1,81 @@
+import { makeConfig } from "@lingui/conf"
+import fs from "fs"
+import path from "path"
+import { vueExtractor } from "../vue-extractor"
+import type { ExtractedMessage } from "@lingui/babel-plugin-extract-messages"
+
+/**
+ * Remove origin from snapshot to avoid
+ * issues with snapshot tests failing on CI
+ * @param snapshot
+ */
+function removeOriginFromSnapshot(snapshot: ExtractedMessage[]) {
+  snapshot.forEach((entry) => {
+    if ("origin" in entry) {
+      delete entry.origin
+    }
+  })
+}
```

Comment:
origins, and working with sourcemaps is an important part of the extractor. You could not just ignore the, in tests. If you have problems with windows path or with diffrent root in CI, it's better to normalize them (cut path from root)